integration/build.gradle

```diff
@@ -3,6 +3,29 @@ repositories {
     maven { url 'http://repository.jboss.org/nexus/content/groups/public' }
 }
 
+project.configurations {
+    integration {
+        extendsFrom project.configurations.testRuntime
+        description = 'Dependencies for integration tests'
+        transitive = true
+        visible = true
+    }
+    alpnboot
+}
+
+def getAlpnVersion() {
```

Comment:
This is very annoying and quite important for anyone who tries to run our integration tests... This does not concern actual deployment in any way. That's why it is only needed in 'integration' module.